src/v/cluster/feature_backend.cc

```diff
@@ -39,7 +42,31 @@ feature_backend::apply_update(model::record_batch b) {
             });
       });
 
+    // Updates to the feature table are very infrequent, usually occurring
+    // once during an upgrade.  Snapshot on every write, so that as soon
+    // as a feature has gone live, we can expect that on subsequent startup
+    // it will be live from early in startup (as soon as storage subsystem
+    // loads and snapshot can be loaded)
```

Comment:
Do we incur this cost every time we replay/bootstrap the controller log? Should we gate this on if the feature state has actually changed to active from inactive or something?

EDIT: Just got to the `applied_offset`, makes sense.